src/obb_rules/game.cljc

```diff
@@ -30,11 +30,13 @@
 
 (defn create
   "Creates a game for a given stash"
-  [stash]
-  (let [board (board/create-board)
-        board1 (board/set-stash board :p1 stash)
-        board2 (board/set-stash board1 :p2 stash)]
-    (assoc board2 :state :deploy)))
+  ([stash]
+    (create stash stash))
+  ([stash1 stash2]
+   (-> (board/create-board)
+       (board/set-stash :p1 stash1)
+       (board/set-stash :p2 stash2)
+       (assoc :state :deploy))))
```

Comment:
I changed this interface to be slightly different in my branch, but it also supports passing in a different stash for each player. Just outlining this here so you keep an eye on that when I open my PR.